### ISSUE TITLE
Rails/TimeZone cop complaints about Time.now.in_time_zone

### ISSUE THREAD

**Issue by sstralenia** [bbatsov/rubocop#2342]
I have such line in my code:
```ruby
Time.now.in_time_zone(@time_zone)
```

And rubocop complaints:
```ruby
app/controllers/api/timelines_controller.rb:51:10: C: Rails/TimeZone: Do not use Time.now without zone. Use Time.zone.now instead.
    Time.now.in_time_zone(@time_zone)
         ^^^
```

Is it a bug in rubocop? Or may be I make something wrong?

**Comment by mikegee** [comment]
I'm not sure if that is a bug or not, but it is easy to avoid. Try:

```ruby
Time.current.in_time_zone(@time_zone)
```

**Comment by sstralenia** [comment]
@mikegee Yes, of course, I know about ```#current```, but I want to figure out in this concrete situation.

**Comment by mikegee** [comment]
I can't get `Time.now.in_time_zone(@time_zone)` to give the wrong answer with some manual testing.

I don't recommend changing it though. Most of the cops in Rubocop are trying to get folks to follow a guideline, not to just avoid buggy code. Allowing this one usage of `Time.now` goes against the principle of it.

**Comment by alexdowad** [comment]
Looks like this can be closed.
